fastlane/Gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-gem "fastlane"
+gem "fastlane", "2.141.0"
```

Comment:
Any reason we're pinning the fastlane version? Just curious.

Reply:
Yes, the newer version of fastlane has a issue with AWS S3, as aws made an update from v2 to v3 and by upgrading fastlane is causing a dependency mismatch, so will be pinned until that is resolved.